lib/utils/file.js

```diff
@@ -0,0 +1,17 @@
+// Based on https://github.com/kevva/executable
+// Since this module doesn't expose the function to check stat mode only,
+// his logic is pasted here.
+// Opened an issue and a pull request about it,
+// to maybe switch to module in the future:
+// Issue: https://github.com/kevva/executable/issues/9
+// PR: https://github.com/kevva/executable/pull/10
+
+export function isExecutable (fileStat) {
+  if (process.platform === 'win32') return true;
```

Comment:
Don't know how the windows implementation will work, if it will use the new windows-bash or something, maybe just wait till we can run hyperterm on windows and take care of that then 👍  I just thought it was weird that it always returned true in windows (which it also does in the [original implementation](https://github.com/kevva/executable/blob/master/index.js#L7))